dashboard/test/models/contact_rollups_pardot_memory_test.rb

```diff
@@ -0,0 +1,76 @@
+require 'test_helper'
+require 'cdo/contact_rollups/v2/pardot'
+
+class ContactRollupsPardotMemoryTest < ActiveSupport::TestCase
+  test 'update_from_new_contacts creates one row per email' do
+    response = Nokogiri::XML::Builder.new do |xml|
```

Comment:
Rubocop really doesn't like this syntax, but its suggested fix is impossible to read. Not sure what to do.